flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/ExecutionGraphCheckpointCoordinatorTest.java

```diff
@@ -144,11 +228,14 @@ private ExecutionGraph createExecutionGraphAndEnableCheckpointing(
 			.setJobGraph(new JobGraph(jobVertex))
 			.setRpcTimeout(timeout)
 			.setAllocationTimeout(timeout)
+			.setIoExecutor(Executors.directExecutor())
 			.build();
 
+		final ExecutionJobVertex executionJobVertex = executionGraph.getVerticesTopologically().iterator().next();
+
 		CheckpointCoordinatorConfiguration chkConfig = new CheckpointCoordinatorConfiguration(
-			100,
-			100,
+			1000000,
+			1000000,
```

Comment:
Why did you need to bump those timeouts?

Reply:
Yes, the timeout is one of the reasons. There is another reason that I don't want the periodic checkpoint triggered during testing. It would disturb the verification.